servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -898,8 +898,8 @@ private boolean isInMemory(BlockInfo blockInfo) {
    *
    * @param path the path of the directory
    * @param options method options
-   * @return a {@link InodeTree.CreatePathResult} representing the modified inodes and created
-   *         inodes during path creation
+   * @return an {@link tachyon.master.file.meta.InodeTree.CreatePathResult} representing the
```

Comment:
Will the generated java doc say `tachyon...` or `InodeTree...` or `CreatePathResult`?